docs/experimental/swap.md

```diff
@@ -0,0 +1,145 @@
+---
+title: SWAP Bandwidth Accounting
+id: swap
+---
+
+Underpinning Swarm is a set of accounting protocols that have been developed and prototyped during over 5 years of R&D. Your Beta Bee node contains functionality giving it the ability to keep track of bandwidth exchanges with all it's peers, issue and cache cheques using smart contracts which live on EVM compatible blockchain. Currently we are in the testing phase using the Goerli testnet. We would love for you to get involved, help us try out these incentives and maybe even earn some GBZZ!
+
+:::info
+Learn more about how SWAP and the other accounting protocols work by reading the [Book of Swarm](https://swarm-gateways.net/bzz:/latest.bookofswarm.eth/the-book-of-swarm.pdf).
+:::
+
+To enable SWAP mode, you must include configuration paramaters `--swap-enabled`, and also a valid [Goerli Testnet](https://goerli.net/) RPC endpoint. An example of this is provided at [rpc.slock.it/goerli](https://rpc.slock.it/goerli), you may sign up for a free account at [Infura](https://infura.io/) run your [own node](https://github.com/goerli/testnet).
```

Comment:
Maybe move the own node part to the front. This is really the mode that should be encouraged, especially for goerli where this is very feasable (fast sync can be completed in <1h).